cmd/xcode.go

```diff
@@ -61,6 +62,40 @@ func initExportOutputDir() (string, error) {
 	return absExportOutputDirPath, nil
 }
 
+// findProject scans the directory for Xcode Project (.xcworkspace / .xcodeproject) file first
+// If can't find any, ask the user to drag-and-drop the file
+func findProject() (string, error) {
+	var projpth string
+
+	projPaths, err := scanForProjectFiles(iOSProjectType)
+	if err != nil {
+		log.Printf("Failed: %s", err)
+		fmt.Println()
+
+		log.Infof("Provide the project file manually")
+		askText := `Please drag-and-drop your Xcode Project (` + colorstring.Green(".xcodeproj") + `) or Workspace (` + colorstring.Green(".xcworkspace") + `) file, 
+the one you usually open in Xcode, then hit Enter.
+(Note: if you have a Workspace file you should most likely use that)`
+		projpth, err = goinp.AskForPath(askText)
+		if err != nil {
+			return "", fmt.Errorf("failed to read input: %s", err)
+		}
```

Comment:
I would return projpath here, instead of else.

Reply:
Fixed in:
https://github.com/bitrise-tools/codesigndoc/pull/95/commits/a8188d992bf9be6815dac6247f057ee110999c50